kedro/framework/cli/cli.py

```diff
@@ -87,8 +87,8 @@ def cli():
     """Kedro is a CLI for creating and using Kedro projects
     For more information, type ``kedro info``.
 
-    When inside a Kedro project (created with `kedro new`) commands from
-    the project's `cli.py` file will also be available here.
+    When inside a Kedro project (created with ``kedro new``) commands
```

Comment:
What's the difference between single vs. double backticks?

Reply:
It depends. I usually refer to [this StackOverflow answer](https://stackoverflow.com/a/57302493/1093967), where you see that there's no difference in Markdown, but some difference in interpolation using reStructuredText.

However, in this case, it literally gets rendered as backticks on the command line, so might as well have a common user experience:

![image](https://user-images.githubusercontent.com/14007150/103806514-2af97100-5023-11eb-947f-e62672588add.png)

Reply:
Thanks for explaining! That's definitely a good point 🙂